composer/utils/collect_env.py

```diff
@@ -282,38 +270,38 @@ def get_torch_env() -> str:
 
 # Composer environment information string output format
 _COMPOSER_ENV_INFO_FORMAT = """
-Composer version: {composer_version}
-Composer commit hash: {composer_commit_hash}
-Host processor model name: {host_processor_model_name}
-Host processor core count: {host_processor_core_count}
-Number of nodes: {node_world_size}
-Accelerator model name: {accelerator_model_name}
-Accelerators per node: {local_world_size}
+Composer Version: {composer_version}
+Composer Commit Hash: {composer_commit_hash}
+CPU Model: {cpu_model}
+CPU Count: {cpu_count}
+Number of Nodes: {node_world_size}
+GPU Model: {gpu_model}
+GPUs per Node: {local_world_size}
+GPU Count: {num_gpus}
 CUDA Device Count: {cuda_device_count}
 """.strip()
 
 
 # Get composer environment info as a dictionary
 def get_composer_env_dict() -> dict:
     """Query Composer pertinent system information as a dict."""
-    mutable_dict = ComposerEnv(
-        composer_version=get_composer_version(),
-        composer_commit_hash=get_composer_commit_hash(),
-        host_processor_model_name=get_host_processor_name(),
-        host_processor_core_count=get_host_processor_cores(),
-        node_world_size=get_node_world_size(),
-        accelerator_model_name=get_accel_model_name(),
-        local_world_size=get_local_world_size(),
-        cuda_device_count=get_cuda_device_count(),
-    )._asdict()
-    return mutable_dict
+    return {
+        'composer_version': get_composer_version(),
+        'composer_commit_hash': get_composer_commit_hash(),
+        'cpu_model': get_cpu_model(),
+        'cpu_count': get_cpu_count(),
+        'num_nodes': get_node_world_size(),
+        'gpu_model': get_gpu_model(),
+        'local_world_size': get_local_world_size(),
```

Comment:
should be `num_gpus_per_node`
```suggestion
        'local_world_size': get_local_world_size()
```